imagefiles/dockcross

```diff
@@ -142,6 +146,11 @@ FINAL_CONFIG=${ARG_CONFIG-${DOCKCROSS_CONFIG-$DEFAULT_DOCKCROSS_CONFIG}}
 # Set the docker image
 FINAL_IMAGE=${ARG_IMAGE-${DOCKCROSS_IMAGE-$DEFAULT_DOCKCROSS_IMAGE}}
 
+# Handle specicial update command
+[[ "$special_update_command" == "update" ]] && command:update && exit
```

Comment:
How about `&& exit` -> `; exit $?` to always exit and pass along the most recent return code?